src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/factory/eventSendReport.hpp

```diff
@@ -59,7 +54,10 @@ class TEventSendReport final : public AbstractHandler<std::shared_ptr<TScanConte
 {
 private:
     std::shared_ptr<TReportDispatcher> m_reportDispatcher;
-
+    static constexpr long HTTP_CONTENT_LENGTH = 413;
```

Comment:
If it is an error, I would prefer to change the name of the variable to something more descriptive. Example:
```suggestion
    static constexpr long HTTP_PAYLOAD_TOO_LARGE = 413;
```